main.tf

```diff
@@ -90,26 +94,28 @@ module "iap_tunneling" {
 }
 
 resource "google_service_account_iam_binding" "bastion_sa_user" {
-  service_account_id = google_service_account.bastion_host.id
+  count              = var.service_account_email == "" ? 1 : 0
+  service_account_id = local.service_account_id
   role               = "roles/iam.serviceAccountUser"
   members            = var.members
 }
 
 resource "google_project_iam_member" "bastion_sa_bindings" {
-  for_each = toset(compact(concat(
+  for_each = var.service_account_email == "" ? toset(compact(concat(
```

Comment:
Please add this to a local now that it has a conditional

Reply:
done